src/utils/constants.js

```diff
@@ -8,7 +8,7 @@ export const green = '#4ddaba';
 export const blue = '#1f90db';
 export const gray = '#9b9b9b';
 export const lightgray = '#d0d0d0';
-export const graphBkgd = '#f6f5f5';
+export const graphBkgd = '#f0f2f5';
```

Comment:
I think you can use basic colors as strings such as
`background: 'red'`

Unless you are specifying your own special red. If that's the, you may want to rename to red to something else

For scenarioColors, use the const you've already defined, and if not defined, you should define them and use them.
`export const scenarioColors = [green (or 'green'), '#b6f5e7','#19b18e', '#e0fdf7', '#008769', '#acdacf']`